enterprise/api/license.go

```diff
@@ -1,17 +1,58 @@
 package api
 
+import (
+	"fmt"
+	"time"
+
+	"github.com/bytebase/bytebase/api"
+)
+
+// validPlans is a string array of valid plan types.
+var validPlans = []api.PlanType{
+	api.TEAM,
```

Comment:
nit: you could use a map to save a for-loop. Since we only have two plans, it's okay to keep the current way.

validPlans = map[api.PlanType]bool{
  api.TEAM: true,
  api.ENTERPRISE: true,
}

Reply:
I plan to implement a function maybe named `ValidSubscriptionPlan` on `PlanType` later, thus we can get rid of these variables and keep the logic inside `plan.go`.